rosbag2_storage/src/rosbag2_storage/ros_helper.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rosbag2_storage/ros_helper.hpp"
+
+#include <memory>
+#include <string>
+
+#include "rcutils/logging_macros.h"
+#include "rcutils/types.h"
+
+namespace rosbag2_storage
+{
+
+std::shared_ptr<rcutils_char_array_t>
```

Comment:
Yes and no. In the end, we want a `SerializedBagMessage`, but since this function only really fills the `rcutils_char_array_t` part, I think it's cleaner to just provide this.